docs/reference/symbolic.rst

```diff
@@ -0,0 +1,65 @@
+
+.. _symbolic_math:
+
+=============
+Symbolic Math
+=============
+
+You can construct a model, for example a CQM, from symbolic math, which is especially
+useful for learning and testing with small problems.
+
+dimod supports binary and integer variables:
```

Comment:
You kind of say this below with `Such variables are constructed as either...` but IMO it's an extremely key point that these are one-variable models, not variables.